config-templates/module_perun.php

```diff
@@ -272,4 +272,13 @@
             ],
         ],
     ],
+
+    'warning_test_sp_config' => [
+        'text' => [
+            'en' => '<h3>Warning - service in test environment</h3>
+                     <p>Service is in the test environment.<br class="spacer"/>Hit the continue button.</p>',
+            'cs' => '<h3>Varování - testovací služba</h3>
+                     <p>Služba je v testovacím režimu.<br class="spacer"/>Pokračujte zmáčknutím tlačítka.</p>',
```

Comment:
Maybe we can split the text into header and info to fix the structure.

Reply:
Refactored.